app/services/workflow_errors_reporter.rb

```diff
@@ -0,0 +1,50 @@
+require 'faraday'
+require 'retries'
+# send errors to preservationWF workflow for an object via ReST calls.
+class WorkflowErrorsReporter
+
+  def self.update_workflow(druid, process_name, error_message)
+    response = http_workflow_request(druid, process_name, error_message)
+
+    # withouth this `if response` I was getting: 
+    # got: (#<NameError: uninitialized class variable @@connection in WorkflowErrorsReporter>)
+    # had to add this condition to get around it. This is for the test:
+    # workflow_errors_reporter_spec.rb:41
+    if response 
+      if response.status == 204
+        Rails.logger.debug("Server responded with status 204, PUT request went through")
+      else
+        # There is not `response.status == 400` because double logging will occur
+        # with the rescue. The rescue will handle the status 400.
+        Rails.logger.warn("PUT request did not go through")
```

Comment:
I think you want a little more context here:  what put request?  for what?  how about  "#{druid} - unable to update workflow: #{response.body}" ??